app/src/main/java/com/claraberriel/mvpparking/mvp/model/ReleaseModel.java

```diff
@@ -0,0 +1,47 @@
+package com.claraberriel.mvpparking.mvp.model;
+
+
+import com.claraberriel.mvpparking.entities.Parking;
+import com.claraberriel.mvpparking.entities.Reservation;
+
+import java.util.Date;
+
+public class ReleaseModel {
+
+    private final Parking parking;
+
+    public ReleaseModel(Parking parking) {
+        this.parking = parking;
+    }
+
+    public Parking getParking() {
+        return parking;
+    }
+
+    public int getParkingLotNumber(String parkingNumber) throws IllegalArgumentException {
+        int result = Integer.parseInt(parkingNumber);
+        if (result <= 0) {
+            throw new IllegalArgumentException();
+        }
+        return result;
+    }
+
+    public boolean checkIfAnyReservationExists() throws IllegalArgumentException {
+        if (getParking().getReservations().size() == 0) {
+            throw new IllegalArgumentException();
+        }
+        return true;
+    }
```

Comment:
This check could be done in parkingRelease(). This method receives no arguments so you shouldn't throw an IllegalArgumentException. You should define your own custom exception.